prov/rxm/src/rxm.h

```diff
@@ -359,7 +361,8 @@ enum {
 	rxm_ctrl_atomic,
 	rxm_ctrl_atomic_resp,
 	rxm_ctrl_credit,
-	rxm_ctrl_rndv_write_ack
+	rxm_ctrl_rndv_write_ack,
+	rxm_ctrl_rndv_done
```

Comment:
Having an ack and a done message is confusing.  Why are there 2 messages?  What is ack acking?

Reply:
the ack is acknowledging the rndv request and providing the sender with reciever side MR information.  The done message is the sender telling the receiver all data has been sent completely

Reply:
We need to fixup the names here.  There's rndv, rndv_ack (which is read done?), rdvn_write_ack (but no rndv_write), and rndv_done (which only applies to write?).  Maybe?

rdnv_req
rdnv_rd_done
rdnv_wr_done
rdnv_wr_data